src/core/janeway_global_settings.py

```diff
@@ -679,3 +679,37 @@ def __len__(self):
         "| a11ycheck ltr rtl | code",
     **TINYMCE_CLIPBOARD_CLEANER,
 }
+
+JATS_ARTICLE_TYPES = (
+    ("abstract", ("Abstract")),
+    ("addendum", ("Addendum")),
+    ("announcement", ("Announcement")),
+    ("article-commentary", ("Article Commentary")),
+    ("book-review", ("Book Review")),
+    ("books-received", ("Books Received")),
+    ("brief-report", ("Brief Report")),
+    ("calendar", ("Calendar")),
+    ("case-report", ("Case Report")),
+    ("collection", ("Collection")),
+    ("correction", ("Correction")),
+    ("discussion", ("Discussion")),
+    ("dissertation", ("Dissertation")),
+    ("editorial", ("Editorial")),
+    ("in-brief", ("In-Brief")),
+    ("introduction", ("Introduction")),
+    ("letter", ("Letter")),
+    ("meeting-report", ("Meeting Report")),
+    ("news", ("News")),
+    ("obituary", ("Obituary")),
+    ("oration", ("Oration")),
+    ("partial-retraction", ("Partial Retraction")),
+    ("product-review", ("Product Review")),
+    ("rapid-communication", ("Rapid Communication")),
+    ("reply", ("Reply")),
+    ("reprint", ("Reprint")),
+    ("research-article", ("Research Article")),
+    ("retraction", ("Retraction")),
+    ("review-article", ("Review Article")),
+    ("translation", ("Translation")),
+    ("other", "Other")
+)
```

Comment:
Is it worth flagging these for translation as you had previously done?

Reply:
Unsure whether we want to load translation in the global settings file? Also: these are fixed in English for the jats type... Unsure we should allow translation as the user needs to see what is entered in the field.

Reply:
doh

Reply:
sounds good